pkg/extender/extender_test.go

```diff
@@ -2613,6 +2618,55 @@ func skipScoringForWindowsPods(t *testing.T) {
 		prioritizeResponse)
 }
 
+// Verify scoring is skipped entirely if Pod uses a volume with winshare "true"
+func skipScoringForDirectAccessPods(t *testing.T) {
+	nodes := &v1.NodeList{}
+	nodes.Items = append(nodes.Items, *newNode("node1", "node1", "192.168.0.1", "rack1", "a", "zone1"))
+	nodes.Items = append(nodes.Items, *newNode("node2", "node2", "192.168.0.2", "rack1", "a", "zone1"))
+	nodes.Items = append(nodes.Items, *newNode("node3", "node3", "192.168.0.3", "rack1", "a", "zone1"))
+	nodes.Items = append(nodes.Items, *newNode("node4", "node4", "192.168.0.4", "rack1", "a", "zone1"))
+	nodes.Items = append(nodes.Items, *newNode("node5", "node5", "192.168.0.5", "rack1", "a", "zone1"))
+	nodes.Items = append(nodes.Items, *newNode("node6", "node6", "192.168.0.6", "rack1", "a", "zone1"))
+
+	if err := driver.CreateCluster(6, nodes); err != nil {
+		t.Fatalf("Error creating cluster: %v", err)
+	}
+	pod := newPod("directAccessVolumePod", defaultNamespace, map[string]bool{"directAccessVolume": false})
+
+	regularVolumeProvNodes := []int{0, 1, 2}
+	if err := driver.ProvisionVolume("directAccessVolume", regularVolumeProvNodes, 3, map[string]string{"backend": "pure-block"}, false, false, true, ""); err != nil {
+		t.Fatalf("Error provisioning volume: %v", err)
+	}
+
+	if err := driver.UpdateNodeStatus(5, volume.NodeOffline); err != nil {
+		t.Fatalf("Error setting node status to Offline: %v", err)
+	}
+
+	filterResponse, err := sendFilterRequest(pod, nodes)
+	if err != nil {
+		t.Fatalf("Error sending filter request: %v", err)
+	}
+	// All nodes exluding the offline nodes will be returned as scheduler is disabled for Windows Pods
+	verifyFilterResponse(t, nodes, []int{0, 1, 2, 3, 4}, filterResponse)
```

Comment:
Couldn't understand how this is related to Windows pods. A comment here or maybe in PR description would help.

Reply:
It's a wrong comment line. Fixed it.